packages/integration/src/gerrit/core.ts

```diff
@@ -67,9 +69,15 @@ export function parseGerritGitilesUrl(
   }
   const project = trimStart(parts.slice(0, projectEndIndex).join('/'), '/');
 
-  const branchIndex = parts.indexOf('heads');
+  let branchIndex = parts.indexOf('heads');
```

Comment:
The "parseGitilesUrlRef" function can already parse urls containing tags, why not use that? Then we don't up with the reading a "tag" from a variable called "branch".

Reply:
Acknowledged!